api_tests/base/test_views.py

```diff
@@ -43,8 +43,9 @@
         if hasattr(patt, 'url_patterns'):
             # Namespaced list of patterns
             for subpatt in patt.url_patterns:
-                VIEW_CLASSES.append(subpatt.callback.cls)
-        else:
+                if hasattr(subpatt.callback, 'cls'):
```

Comment:
This is another line I can't quite tell why this is needed, but I'm sure there's a good reason for it. Maybe use a named function or throw a comment in there.

Reply:
another "allow view functions to exist" -- will comment